src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -84,7 +83,7 @@ export class ExamUpdateComponent implements OnInit {
 
     get isValidConfiguration(): boolean {
         const examConductionDatesValid = this.isValidVisibleDate && this.isValidStartDate && this.isValidEndDate;
-        const examReviewDatesValid = this.isValidPublishResultsDate && this.isValidExamStudentReviewStart && this.isValidExamStudentReviewEnd;
+        const examReviewDatesValid = this.isValidPublishResultsDate && this.isValidExamStudentReviewStart && this.isValidExamStudentReviewEnd && !this.exam.reviewPhaseError;
```

Comment:
with the introduced changes, isn't `this.isValidExamStudentReviewEnd` redundant or kind of overwritten? See my general comment on this PR regarding to validation concepts. 
Also, I would not call the flag `reviewPhaseError`, I would stick to the concept of `isValidXYZ`.
Wouldn't it be possible to include the logic of validating having an end date when a start date is set into the flag `isValidExamStudentReviewEnd`, i.e. [here](https://github.com/ls1intum/Artemis/pull/3875/files#diff-cef0731e11f6e67dcaef57890eff1931526a2ad05bed9d7af957b5e8122253fbR130).

Reply:
As I started creating the logic, I tried to stick to the solution that is already used for exercises in order to check for a valid assessment due date. But yes, this leads to duplicates. I changed it in order to allocate the logic in the existing method `isValidExamStudentReviewEnd()`. Feel free to have a look again!